dev-tools/startup/discovery-provider.sh

```diff
@@ -5,6 +5,10 @@ if [[ "$audius_db_url" == "" ]]; then
     export audius_db_url_read_replica="postgresql+psycopg2://postgres:postgres@db:5432/discovery_provider_${replica}"
 fi
 
+if [[ "$audius_redis_url" == "" ]]; then
+    export audius_redis_url="redis://audius-protocol-discovery-provider-redis-${replica}:6379/00"
```

Comment:
instead of defaulting DN to connect to localhost redis, point to its matching Redis replica